### PR TITLE
Issue 14917: Clarify JWT message about JWE requirements and feature level required

### LINKED ISSUE #14917: Enhance error handling when JWE's are used with mpJwt-1.1
Feature mpJwt-1.2 is adding support for JWE's (mpJwt-1.1 only supports JWS's)
If we only have mpJwt-1.1 enabled and try to consume a JWE, we receive this message:
``CWWKS6063E: The token in the request is not in JSON Web Signature (JWS) format, but only tokens that are in JWS format are accepted. If the token is in JSON Web Encryption (JWE) format, configure the mp.jwt.decrypt.key.location MicroProfile Config property or the keyManagementKeyAlias attribute in the [mpJwt_1] JWT consumer configuration.``
1) if we use a JWE token with only the 1.1 feature and no decrypt config in either the server.xml or in mp config properties, we should get a message stating that JWE's are not supported.
2) if we use a JWE token with the 1.1 feature and do have decrypt config in either the server.xml or in mp config properties, we should get a message stating that JWE's are not supported - currently, we get the CWWKS6063E message (which is really confusing because I did specify what the message tells you to...  (The feature being 1.1 is what causes the null value for the key loc)
3) We may want to log a warning if we find the decrypt config values (in either the server.xml or the mp config properties file)

To reproduce: 
1) configure an app to use mpJwt - specify mpJwt-1.1 as the feature
2) create a JWE token (instead of a JWS token) and try to access your protected app using the JWE Token 
3) additionally,  add ``keyManagementKeyAlias=<decryptKeyAlias>`` to your mpJwt config in server.xml, or include mp.jwt.decrypt.key.location=<privateKey pem file> in the mp config properties file in your app or via env or system properties.

### PR DESCRIPTION
**Pull request by ayoho**
Resolves #14917

### CODE REVIEW

**Comment by ayoho** [comment]
#build

File: dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops [hunk 1]
```diff
@@ -371,8 +371,8 @@ DECRYPT_KEY_LOCATION_INLINE_KEY.useraction=Use a relative path or URL as the val
 # Do not translate "mp.jwt.decrypt.key.location", "keyManagementKeyAlias", "Content Encryption Key"
 # 0=JWT consumer config ID
 JWS_REQUIRED_BUT_TOKEN_NOT_JWS=CWWKS6063E: The token in the request is not in JSON Web Signature (JWS) format, but the [{0}] JWT consumer only accepts tokens that are in JWS format.
-JWS_REQUIRED_BUT_TOKEN_NOT_JWS.explanation=The token in the request might be malformed or might be in JSON Web Encryption (JWE) format. Tokens must be in JWS format if the JWE decryption key location is not configured.
-JWS_REQUIRED_BUT_TOKEN_NOT_JWS.useraction=To accept JWE tokens, configure the mp.jwt.decrypt.key.location MicroProfile Config property or the keyManagementKeyAlias attribute in the JWT consumer configuration. Otherwise, ensure that only JWS tokens are sent to this resource.
+JWS_REQUIRED_BUT_TOKEN_NOT_JWS.explanation=The token in the request might be malformed or might be in JSON Web Encryption (JWE) format. Tokens must be in JWS format if the JWE decryption key location is not configured. If using the MicroProfile JWT feature, the 1.2 version or beyond of the feature must be configured to accept tokens in JWE format.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops:374]: Change,
"If using the MicroProfile JWT feature, the 1.2 version or beyond of the feature must be configured to accept tokens in JWE format."
to,
"If you use the MicroProfile JWT feature, the 1.2 version or beyond of the feature must be configured to accept tokens in JWE format."

File: dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops [hunk 2]
```diff
@@ -371,8 +371,8 @@ DECRYPT_KEY_LOCATION_INLINE_KEY.useraction=Use a relative path or URL as the val
...
+JWS_REQUIRED_BUT_TOKEN_NOT_JWS.useraction=To accept JWE tokens, configure the mp.jwt.decrypt.key.location MicroProfile Config property or the keyManagementKeyAlias attribute in the JWT consumer configuration. If using the MicroProfile JWT feature, ensure that the 1.2 version or beyond is configured. Otherwise, ensure that only JWS tokens are sent to this resource.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops:375]: Change,
"If using the MicroProfile JWT feature, ensure that the 1.2 version or beyond is configured. "
to,
"If you use the MicroProfile JWT feature, ensure that the 1.2 version or beyond is configured. "

**Review by ManasiGandhi — CHANGES_REQUESTED** [review]
Some edit suggestions.

**Comment by andrea-pichler** [comment]
Completed the requested L2 message review

File: dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops (hunk 1 above)
Comment (ayoho) [reply dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops]: Both usages updated.

**Review by ManasiGandhi — APPROVED** [review]

**Comment by ayoho** [comment]
#run-libby-bot

**Pull request merged**